spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierResolver.java

```diff
@@ -51,37 +51,30 @@ public class TypeQualifierResolver {
     static final ClassDescriptor elementTypeDescriptor = DescriptorFactory
             .createClassDescriptor(ElementType.class);
 
-    static final ClassDescriptor androidNullable = DescriptorFactory.createClassDescriptor("android/support/annotation/Nullable");
-
-    static final ClassDescriptor androidNonNull = DescriptorFactory.createClassDescriptor("android/support/annotation/NonNull");
```

Comment:
`private static final String ANDROID_SUPPORT_ANNOTATION = "android/support/annotation/";`

This line of code still uses the code above on line 54

Reply:
I think the small repetition is fine here, and IMO extracting the package as a constant would be less readable.
Not sure if you realized but you're commenting on the deleted code (highlighted in red)